wfm.c

```diff
@@ -545,6 +545,10 @@ void cfgload(void) {
     if(strlen(cfg.tagline)>2) cfg.tagline[strlen(cfg.tagline)-1]='\0';
     if(strlen(cfg.favicon)>2) cfg.favicon[strlen(cfg.favicon)-1]='\0';
     
+    // remove trailing slash from homedir (causes problems with basename checks)
+    if(cfg.homedir[strlen(cfg.homedir)-1] == '/')
```

Comment:
also add check if this is valid address (> 0), eg

if(strlen(cfg.homedir)-1 && .....

otherwise it may crash trying to access -1 array index ;)